backend/tests/integration/conftest.py

```diff
@@ -345,9 +234,5 @@ def app_cli_runner(appctx):
 
 
 @pytest.fixture(scope="function")
-def redis(base_app):
-    redis_url = current_app.config.get("CACHE_REDIS_URL")
-    r = StrictRedis.from_url(redis_url, decode_responses=True)
-    r.flushall()
-
-    yield r
+def app_clean(base_app, db, es_clear, vcr_config):
```

Comment:
I thought we ``es_clear`` was slow and we'll not add it to the tests, does it slow down the tests a lot?

Reply:
At the end it added 5 seconds for all 200 changed tests from ~65 seconds to ~70 seconds on my machine 
By average it's +0.025s per test

Reply:
that's cool, great work! can we test if for all tests? also if we use at the end only ``es_clear`` I'd call the fixture just ``app``

Reply:
To make it work with all the tests I have to go through all tests. You can notice that many fixtures like for example `create_record` wont' be a fixtures anymore. So I have to go every test, change fixtures and import all required ex-fixtures from test utils ;) This will take some time and this is why I asked for your comments as I don't want to go through all tests more than once if something should be changed or done differently in your opinion.